src/main/java/stirling/software/SPDF/config/security/database/DatabaseConfig.java

```diff
@@ -16,7 +18,9 @@
 
 @Slf4j
 @Getter
+@Lazy
 @Configuration
+@ConditionalOnProperty(name = "premium.proFeatures.database", havingValue = "true")
```

Comment:
This isnt correct, we allow users without premium to use H2 and user database etc

Reply:
Should be based on enabled login